### ISSUE TITLE
kuma-cni runs into permission issues when installed in a different namespace than "kube-system"

### ISSUE THREAD

**Issue by jijiechen** [kumahq/kuma#9655]
### What happened?

When kuma-cni is installed in a a different namespace than "kube-system",  it doesn't work correctly and I can see a warning in the pod description:

```
ERRORED: error configuring pod [kuma-demo/kuma-demo-backend-v0-7896cd8486-qnkl8] networking: [kuma-demo/kuma-demo-backend-v0-7896cd8486-qnkl8/e8e56a75-241f-454c-812c-d2fd00e3fe6a:kuma-cni]: error adding container to network "kuma-cni": pod excluded - error getting pod info: failed to get pod data: pods "kuma-demo-backend-v0-7896cd8486-qnkl8" is forbidden: User "system:serviceaccount:openshift-sdn:kuma-cni" cannot get resource "pods" in API group "" in the namespace "kuma-demo
```

In my case, `kuma-cni` was installed in the `openshift-sdn` namespace.

There is an incorrect service namespace reference with the helm chart template.

**Comment by slonka** [comment]
There is another user that wants to run cni in a another namespace.

**Comment by jijiechen** [comment]
It's fixed in 2.7.
